bsdfdisk/edit_part.cgi

```diff
@@ -49,10 +49,14 @@ if ($canedit) {
 	print &ui_table_row($text{'part_type'},
 		&ui_select("type", $part->{'type'},
 			   [ &list_partition_types() ], 1, 0, 1));
+	print &ui_table_row($text{'part_label'},
+		&ui_textbox("label", $part->{'label'}, 20));
```

Comment:
@karmantyu, remember to maintain indent correctly! We use **tabs** with tab display **size** set to **8**.